cmd/karavictl/cmd/tenant.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+// Copyright © 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
The [YEARS] field should indicated the range [year file was created - year file was last updated]